src/univariate/continuous/chernoff.jl

```diff
@@ -137,11 +138,11 @@ module ChernoffComputations
 
         (y > 0) || throw(DomainError(y, "argument must be positive"))
 
-        cnsty = y^(-1.5)
+        cnsty = inv(sqrt(y^3))
         if (y <= 1.0)
-            return sum([(b[k]*cnsty - a[k]*sqrthalfpi)*y^(3*k) for k=1:length(a)])-sqrthalfpi
+            return sum((b[k]*cnsty - a[k]*sqrthalfpi)*y^(3.0*k) for k in 1:length(a)) - sqrthalfpi
```

Comment:
I don't think it's necessary to change `3` to `3.0`?
```suggestion
            return sum((b[k]*cnsty - a[k]*sqrthalfpi)*y^(3*k) for k in 1:length(a)) - sqrthalfpi
```